core/services/pipeline/task.bridge.go

```diff
@@ -108,9 +113,26 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
 	requestCtx, cancel := httpRequestCtx(ctx, t, t.config)
 	defer cancel()
 
+	var cachedResponse bool
 	responseBytes, statusCode, headers, elapsed, err := makeHTTPRequest(requestCtx, lggr, "POST", URLParam(url), []string{}, requestData, t.httpClient, t.config.DefaultHTTPLimit())
 	if err != nil {
-		return Result{Error: err}, RunInfo{IsRetryable: isRetryableHTTPError(statusCode, err)}
+		if cacheTTL == 0 {
+			return Result{Error: err}, RunInfo{IsRetryable: isRetryableHTTPError(statusCode, err)}
+		}
+
+		lggr.Debugw("Bridge task: request failed, falling back to cache",
```

Comment:
I would rather log the fact we are using a cached value (near line 135), and dump that value as Debug.
For the failure case we have other logging on the line 129.